src/rma/destruction/models.py

```diff
@@ -167,3 +167,24 @@ class Meta:
 
     def __str__(self):
         return f"({self.destruction_list_review}) - ({self.destruction_list_item})"
+
+
+class DestructionListAssignee(models.Model):
+    destruction_list = models.ForeignKey(
+        DestructionList,
+        on_delete=models.CASCADE,
+        related_name="assignees",
+        verbose_name=_("destruction list"),
+    )
+    assignee = models.ForeignKey(
+        "accounts.User", on_delete=models.CASCADE, verbose_name=_("assignee"),
```

Comment:
```suggestion
        "accounts.User", on_delete=models.PROTECT, verbose_name=_("assignee"),
```
Let's prevent (accidentally) deleting users that have played a role in a destruction list.